src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/helper/EnvelopeCreator.java

```diff
@@ -236,4 +256,8 @@ public static OcrData ocrData(Map<String, String> data) {
     private static Instant getInstant() {
         return Instant.parse("2018-06-23T12:34:56.123Z");
     }
+
+    private static Instant getInstant(int year, int month, int day, int hour, int minute, int second) {
+        return Instant.parse(String.format("%d-%02d-%02dT%02d:%02d:%02d.000Z", year, month, day, hour, minute, second));
+    }
```

Comment:
perpaps we can simply pass one parameter (LocalDateTime) instead of 6 and it will be simpler